### ISSUE TITLE
Adding album does not add it, neither Artist

### ISSUE THREAD

**Issue by btTeddy** [lejacobroy/Tidalrr#21]
When adding album through `/actions/addFromUrl`, album seems to be sucesfully added but nowhere to be found

Example log:

```
Importing

Logs
Importing URL: https://listen.tidal.com/album/1225569
Non-Empty Access Token, logging in
Url #1: https://listen.tidal.com/album/1225569
Added album to DB: Hybrid Theory (Bonus Edition)
```

When checking Library/Albums or Library/Artist there is no such album/artis

Adding Artist alone or Playlist works just fine

**Comment by ramonskie** [comment]
you need to trigger scan manually or wait for it to trigger

**Comment by nanoscopic** [comment]
As I explain in #37 , there is a bug in that the system doesn't add the artist needed so the process doesn't complete.

You generally have to add from playlists to get the artists to add currently.

I believe the "left join" changes I suggest in #41 may fix it? I've made too many local changes at this point to know what is fixing the various issues...

Suffice it to say, add at least one track from the album to a playlist and then add the playlist, and then scan. Once the artist is in the DB by doing that adding additional albums directly of that artist will work.

**Comment by ramonskie** [comment]
@nanoscopic it seems @lejacobroy abandoned this project. If you have a working fork. Would we able to use?

**Comment by nanoscopic** [comment]
You could perhaps use it if Tidal wasn't down completely as of an hour ago. 🤷 

Edit: AWS US-East-1 DynamoDB went down. They are claiming a DNS issue so far.

**Comment by ramonskie** [comment]
yeah aws has some issues for the past day already. allot of stuff is slow or non responsive

**Comment by Enrique-Groeneveld** [comment]
@nanoscopic Is there a fork that is usable?

**Comment by nanoscopic** [comment]
The current fork -does- work if you change the client_id it uses to one of the ones other open source Tidal players are now using.

As Tidal just disabled the ones they were all using, Tidal seems to have a dislike for people running open source players so I'm not particularly interested in posting that update or explaining more than just that. I don't use this code to "pirate music", but commenting on the code and where it is broken is the limit of what I feel comfortable with.

As I said in my other issues, it basically only works if you add music via playlists to it.

It does break in some situations ( which I described as well in more tickets )...

I will point out as well that if there is an error and any of its processes are running, mostly it won't stop that process and will continue doing it causing one error after another. That's not great, because it can cause this code to spam Tidal with a series of failing requests.
